src/pages/Categories/CategoriesPage.vue

```diff
@@ -170,7 +183,29 @@ export default {
 	},
 	apollo: {
 		query: allCategoriesPageQuery,
-		preFetch: true,
+		preFetch(config, client) {
+			return client.query({
+				query: allCategoriesPageQuery
```

Comment:
Can we move this to the end of the chain instead of the beginning? It's already a very slow query, determining your exp setting and assignment before hand will allow us to skip fetching it if we plan to redirect.

Reply:
Good call.